### ISSUE TITLE
Connection failure | Auth fail

### ISSUE THREAD

**Issue by david-lev** [tytydraco/SSH#1]
Hi, I have looking for a terminal that allows ssh connection without access to a local terminal (as opposed to JuiceSSH and similar. )
I downloaded the app from Google play but I can't connect.
I have ssh server and I added the rsa public key to ~/.ssh/authorized_keys but he refuses to connect.

By the way, through Termux I manage to connect as usual without problems. 

Galaxy S21 | Android 11
App version 1.0.7

There is anything that I can do to fix this?

**Comment by tytydraco** [comment]
Hey! That's super strange. Could you grab a logcat so I can see what the exact error is? Usually Auth Fail means the username, password, or RSA key are wrong. And just to be sure, you added the RSA key to the SSHD authorized keys list on the server side, correct? Maybe the server needs a SSHD restart to detect the new key list.

**Comment by david-lev** [comment]
for sure. I added the key to the list of keys on the server, where there are 5 more active keys, I started the sshd service and even restarted the server.
The really weird thing is that I now tried to connect from another device (S7 Edge, Android 8) and the connection passed successfully...
I will try to send a logcat later but it may be a limitation on the part of Android system?

And again, thanks for your simple and excellent app, as well as for the quick responses :)
